docs/source/tutorials/configure_auto_optimizer.rst

```diff
@@ -0,0 +1,263 @@
+.. _how_to_configure_auto_optimizer:
+
+[BETA] How To Configure Auto Optimizer
+=======================================
+
+Auto Optimizer is a tool that can be used to automatically search Olive passes combination based on:
+
+1. input model
+2. target device
+3. target precision: `fp32`, `fp16`, `int8`, `int4` and etc.
+4. target evaluation metric: accuracy, latency and etc.
+
+All above information(is called "optimization factors" in this doc) is provided by user through a configuration file now, then run by:
+
+.. code-block::
+
+    python -m olive.workflows.run --config <config_file>.json
+
+
+With the help of Auto Optimizer, for most popular models like bert, deberta and etc., user:
+
+1. DOES NOT need to provide the passes combination manually.
+2. DOES NOT need to write redundant passes configs.
+
+At the same time, user still has the flexibility to specify the accelerator, execution providers and expected precision of output models.
+
+Currently, **auto optimizer supports onnx optimization stack only**. For openvino and snpe, user still need to provide passes combination manually by now.
```

Comment:
Nit: we may need to clarify that openvino is specifically for OpenVINO IR model. ONNXRuntime still have OpenVINO EP (but we don't add this EP to auto optimizer?)

Reply:
We should add OpenVINO EP but it can be done as a follow up PR.

Reply:
Updated to tell the diff between openvino ep and raw openvino IR.